content/en/docs/zero-code/python/troubleshooting.md

```diff
@@ -91,6 +92,105 @@ if __name__ == "__main__":
     app.run(port=8082, debug=True, use_reloader=False)
 ```
 
+### Pre-fork server issues
```

Comment:
For all options besides instrumenting post-fork, there is an issue where the Resources for each worker will be the same. For traces and logs it's OK, but metrics will have [single writer violations](https://opentelemetry.io/docs/specs/otel/metrics/data-model/#single-writer).

We don't need to fix everything in this PR but maybe we can

- recommend instrumenting post-fork as the best option and
- making sure the `process.pid` is detected correctly
- and ideally that each worker has a unique `service.instance.id`

WDYT?

Reply:
I'd like to add the recommendations around `process.pid` and `service.instance.id` in a new PR, if that's ok!